charts/civil-sdt/values.preview.template.yaml

```diff
@@ -12,6 +12,16 @@ java:
   postgresql:
     enabled: true
   keyVaults:
+    civil:
+      secrets:
+        - name: civil-sdt-service-s2s-secret
+          alias: CIVIL_SDT_S2S_KEY
+        - name: civil-sdt-POSTGRES-USER
+          alias: SDT_DB_USERNAME
+        - name: civil-sdt-POSTGRES-PASS
+          alias: SDT_DB_PASSWORD
+        - name: civil-sdt-servicebus-connection-string
+          alias: CIVIL_SDT_SERVICEBUS_CONNECTION_STRING
     civil-sdt:
       disabled: true
```

Comment:
There are two key vaults referenced here - "civil" (line 15) and "civil-sdt" (line 26).  We are no longer using the "civil" key vault, so this needs to be changed to just reference the "civil-sdt" key vault.

Note that enabling the key vault in preview means that the build will fail if a new secret is being created by the pull request. This is because the plan isn't applied in preview build pipeline, so the secret won't exist.